### ISSUE TITLE
Fields are changed just by accessing them

### ISSUE THREAD

**Issue by p3t3r** [grlicaa/AutoNumeric#1]
The autonumeric changes fields if you enter them.
Meaning I just click in the fields, change nothing and when I navigate away from the page, it gives a warning of unsaved changes.

**Comment by grlicaa** [comment]
Hi @p3t3r,

thank you for changing this one, I reproduced error and like documentation says for function 
apex.item("NAME").isChanged() :
> If this function does not work correctly then changes to the plug-in item type value will not be detect and the user will not be warned when they leave the page.

So to fix this I've added additional code on APEX page item :
```javascript
isChanged:function() {
    return (this.node.defaultValue != this.getValue());
},
```

This will be fixed with next version 4.6.0.3 probably in next few days.

**Comment by grlicaa** [comment]
Fixed  4.6.0.3 is out, closing this error. Please let me know if you got any other errors.
